package.json

```diff
@@ -69,7 +68,8 @@
     "intersection-observer": "^0.4.2",
     "pepjs": "^0.4.2",
     "tslib": "~1.8.1",
-    "web-animations-js": "^2.3.1"
+    "web-animations-js": "^2.3.1",
```

Comment:
let's pin these (ideally to matching versions), could you pin the other dependencies too? Don't want any nasty surprises :)

Reply:
Done